ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js

```diff
@@ -88,7 +88,7 @@ const EditGasToolTip = ({
         editGasMode === EDIT_GAS_MODES.SWAPS
       ) &&
       !estimateGreaterThanGasUse ? (
-        <img alt="" src={`./images/curve-${priorityLevel}.svg`} />
+        <img alt="" src={`./images/curve-${priorityLevel}.png`} />
```

Comment:
[REQ] Lets add an alt description using t('').

Reply:
Added the fix for this.